src/openms/include/OpenMS/ANALYSIS/SEQUENCE/NeedlemanWunsch.h

```diff
@@ -0,0 +1,86 @@
+#include <vector>
+#include <OpenMS/DATASTRUCTURES/String.h>
+#include <OpenMS/OpenMSConfig.h>
+
+
+//namespace OpenMS
+//{
+  class OPENMS_DLLAPI NeedlemanWunsch
+  {
+    std::vector<int> adaptedIdentity
```

Comment:
you can move these matrices into the cpp (as global objects). They do not need to be exposed in the header.